xapian-core/cluster/lc.cc

```diff
@@ -91,7 +92,7 @@ LC::cluster(const MSet &mset)
     PSet::iterator it;
     for (unsigned int cnum = 1; cnum <= k; ++cnum) {
 	// Container for new cluster
-	Cluster cluster;
+	Cluster C;
```

Comment:
Local variable names are conventionally lower-case. Also, you should be able to come up with a more useful name than just `C`, even though the obvious one isn't possible.

Reply:
Thanks for pointing this out, I've modified the variable name used in a fixup commit.